front/components/spaces/CreateOrEditSpaceModal.tsx

```diff
@@ -208,112 +213,125 @@ export function CreateOrEditSpaceModal({
   }, [doDelete, handleClose, owner.sId, router, space]);
 
   return (
-    <Modal
-      isOpen={isOpen}
-      onClose={handleClose}
-      title={space ? `Edit ${space.name}` : "Create a Space"}
-      saveLabel={space ? "Save" : "Create"}
-      variant="side-md"
-      hasChanged={
-        !!spaceName &&
-        (!isRestricted || (isRestricted && deduplicatedMembers.length > 0))
-      }
-      isSaving={isSaving}
-      className="flex overflow-visible" // overflow-visible is needed to avoid clipping the delete button
-      onSave={onSave}
-    >
-      <Page.Vertical gap="md" sizing="grow">
-        <div className="flex w-full flex-col gap-y-4 overflow-y-hidden px-1">
-          <div className="mb-4 flex w-full flex-col gap-y-2 pt-2">
-            <Page.SectionHeader title="Name" />
-            <Input
-              placeholder="Space's name"
-              value={spaceName}
-              name="spaceName"
-              onChange={(e) => setSpaceName(e.target.value)}
-            />
-            {!space && (
-              <div className="flex gap-1 text-xs text-element-700">
-                <Icon visual={ExclamationCircleIcon} size="xs" />
-                <span>Space name must be unique</span>
-              </div>
-            )}
-          </div>
-          <div className="flex w-full flex-col gap-y-2 border-t pt-2">
-            <div className="flex w-full items-center justify-between overflow-visible">
-              <Page.SectionHeader title="Restricted Access" />
-              <SliderToggle
-                selected={isRestricted}
-                onClick={() => setIsRestricted(!isRestricted)}
+    <Sheet open={isOpen} onOpenChange={handleClose}>
+      <SheetContent trapFocusScope={false} size="lg">
+        <SheetHeader>
+          <SheetTitle>
+            {space ? `Edit ${space.name}` : "Create a Space"}
+          </SheetTitle>
+        </SheetHeader>
+        <SheetContainer>
+          <div className="flex w-full flex-col gap-y-4">
+            <div className="mb-4 flex w-full flex-col gap-y-2">
+              <Page.SectionHeader title="Name" />
+              <Input
+                placeholder="Space's name"
+                value={spaceName}
+                name="spaceName"
+                onChange={(e) => setSpaceName(e.target.value)}
               />
-            </div>
-            <div className="text-sm font-normal text-element-700">
-              {isRestricted ? (
-                <p>Restricted access is active.</p>
-              ) : (
-                <p>
-                  Restricted access is disabled. The space is accessible to
-                  everyone in the workspace.
-                </p>
+              {!space && (
+                <div className="flex gap-1 text-xs text-element-700">
```

Comment:
Can you add `dark:text-element-700-night` wherever you use `text-element-700` in the file ? 🙏🏻

Reply:
Fully leveraged the `Input` component instead 😇